### PR TITLE
added link to store on product edit page

### PR DESCRIPTION
**Pull request by alejandromangione**
Added link to see product at store.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://alejandromangione-see-product-store.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://alejandromangione-see-product-store.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: backend/app/views/spree/admin/shared/_product_tabs.html.erb [hunk 1]
```diff
@@ -20,5 +20,9 @@
     <%= content_tag :li, class: ('active' if current == :stock) do %>
       <%= link_to_with_icon 'home', Spree.t(:stock), stock_admin_product_url(@product) %>
     <% end if can?(:stock, Spree::Product) %>
+    <%= content_tag :li do %>
+      <%= link_to_with_icon 'eye-open', Spree.t(:see_in_store), product_path, :target => "_blank" %>
```
Comment (BenMorganIO) [inline backend/app/views/spree/admin/shared/_product_tabs.html.erb]: Can you namespace the path as well? Probably call it `spree.product_path`. This way there's no conflicts for apps that might have a `main_app.product_path`, a `product_path` possibly defined in a helper, and the `spree.product_path`. Also, shouldn't an object get passed to the `product_path`? Like `spree.product_path @product`?

Comment (BenMorganIO) [inline backend/app/views/spree/admin/shared/_product_tabs.html.erb]: Also, please use Ruby 1.9 syntax for `:target`. :)

**Comment by alejandromangione** [comment]
Hi Ben, thanks for comments! I applied your suggestions.

**Comment by BenMorganIO** [comment]
:+1:

**Comment by JDutil** [comment]
The product_path route doesn't exist unless your using the frontend as well.  I think to fix the build you'll have to also add the product show route in admin/config/routes.rb

**Comment by BenMorganIO** [comment]
I'm going :-1: on that because some stores don't use a conventional product_show route.

For example, I use a `product_path` as a helper and it works like this:

```ruby
def product_path(product)
  main_app.product_by_slug_path(product.id, product.slug)
end
```

**Comment by JDutil** [comment]
Yea I'm going to close this out and suggest to just implement this in your application.

**Pull request closed without merging**